resources/scripts/components/dashboard/ServerRow.tsx

```diff
@@ -52,6 +52,8 @@ export default ({ server, className }: { server: Server; className: string | und
         alarms.memory = isAlarmState(stats.memoryUsageInBytes, server.limits.memory);
         alarms.disk = server.limits.disk === 0 ? false : isAlarmState(stats.diskUsageInBytes, server.limits.disk);
     }
+    const disklimit = server.limits.disk != 0 ? bytesToHuman(server.limits.disk * 1000 * 1000) : "Unlimited";
```

Comment:
Correct, you would have to refresh the page to see the new value.

Reply:
Doesn't that just depend on whether `server` updates automatically?

Reply:
@schrej Yes, but it is possible server could update automatically, especially from the websocket (since I plan on sending even more events over it). I like to build with the assumption the data could change on the fly later on, rather than have some things working and some things not.

Reply:
And why would that be an issue here? Because of const? The whole function is executed on each render, don't forget that

Reply:
Oh right, totally forgot server was coming from a hook here and will re-render when updated. Nvm.